exp/api/v1alpha3/types.go

```diff
@@ -108,4 +109,6 @@ type AutoScalingGroup struct {
 	// MixedInstancesPolicy        *autoscaling.MixedInstancesPolicy
 	PlacementGroup    string
 	VPCZoneIdentifier []string
+	Instances         []infrav1.Instance    `json:"instances,omitempty"`
+	State             infrav1.InstanceState `json:"instanceState,omitempty"`
```

Comment:
this InstanceState won't line up properly, ASG statuses are slightly different, but I have that coming in the deletion PR